packages/server/dataloader/azureDevOpsLoaders.ts

```diff
@@ -117,6 +117,8 @@ export interface AzureUserInfo {
 export interface AzureAccountProject extends TeamProjectReference {
   userId: string
   teamId: string
+  instanceId: string
+  projectId: string
```

Comment:
We call `allAzureDevOpsProjects` in `fetchAllRepoIntegrations`, which get used by `IntegrationRepoId` which needs the `instanceId` and `projectId`: https://github.com/ParabolInc/parabol/blob/bb6bbf9b7bcb06fb266d036396f1490e7822e31d/packages/client/shared/gqlIds/IntegrationRepoId.ts#L52